styles.css

```diff
@@ -0,0 +1,231 @@
+@media (max-width: 989px) {
+ article {
+    width: 220px;
+    height: 300px;
+    background-color: #A2C4C9;
+    float: left;
+    border: 3px solid black;
+    border-radius: 5px;    
+    margin: 6px;
+    margin-top: 80px;
+    font-size: x-small;
+    box-sizing: border-box;
+    padding: 10px
+    }
+  #first {

```

Comment:
All the articles are different widths with the small screen. Why? You shouldn't need to specify the height either.